From 74196d452a593f9aab375dd4887dac0fdca519e8 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 30 Apr 2015 01:47:00 -0700
Subject: [PATCH] Fix extern inline use for gcc > 4.3 in c99 mode

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 lib/mpi/mpi-inline.h | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/lib/mpi/mpi-inline.h b/lib/mpi/mpi-inline.h
index e2b3985..541e83a 100644
--- a/lib/mpi/mpi-inline.h
+++ b/lib/mpi/mpi-inline.h
@@ -29,8 +29,17 @@
 #ifndef G10_MPI_INLINE_H
 #define G10_MPI_INLINE_H
 
+/* Starting with gcc 4.3 "extern inline" conforms in c99 mode to the
+ * c99 semantics.  To keep the useful old semantics we use an
+ * attribute.
+ */
+
 #ifndef G10_MPI_INLINE_DECL
-#define G10_MPI_INLINE_DECL  extern inline
+#if defined __GNUC_STDC_INLINE__ || defined __GNUC_GNU_INLINE__
+#define G10_MPI_INLINE_DECL extern inline __attribute__ ((__gnu_inline__))
+#else
+#define G10_MPI_INLINE_DECL extern inline
+#endif
 #endif
 
 G10_MPI_INLINE_DECL mpi_limb_t
-- 
2.1.4

